tests/js/client/active-failover/basic.js

```diff
@@ -408,8 +448,10 @@ function ActiveFailoverSuite() {
       });
       suspended = [];
 
-      let upper = checkData(currentLead);
-      print("Leader inserted ", upper, " documents so far");
+      // Wait until all servers report healthy again
+      assertTrue(waitUntilHealthStatusIs(healthyList, []));
```

Comment:
The assertion doesn't hurt, but `waitUntilHealthStatusIs` always returns true (or throws).

Reply:
Will fix it to allow it to return false